# Array of predicted trajectories
std_msgs/Header header
PredictedTrajectory[] trajectories